src/uct/api/uct.h

```diff
@@ -812,10 +813,10 @@ UCT_INLINE_API ucs_status_t uct_ep_put_short(uct_ep_h ep, const void *buffer, un
  * @brief
  */
 UCT_INLINE_API ucs_status_t uct_ep_put_bcopy(uct_ep_h ep, uct_pack_callback_t pack_cb,
-                                             void *arg, size_t length, uint64_t remote_addr,
```

Comment:
Removal of length bothers me. I understand that user pack it in arg together with source addr but it just seems a reasonable argument to pass.